requirements-production-local-forks.txt

```diff
@@ -1,3 +1,3 @@
 git+https://github.com/lepture/mistune-contrib.git#egg=mistune-contrib
-git+https://github.com/pamfilos/invenio-files-rest.git@eos-storage#egg=invenio-files-rest
+git+https://github.com/diegodelemos/invenio-files-rest.git@eos-storage#egg=invenio-files-rest
```

Comment:
Rebased on top of [latest master](https://github.com/inveniosoftware/invenio-files-rest/commits/master). Last release was 1.2.0 and commits on top of it are only related to docs and tests.

Reply:
WRT custom invenio-files-rest hacks, is file download supposed to work? I've tried to run in the production mode:

```console
$ docker-compose build
$ docker-compose up -d
$ docker exec -i -t opendatacernch_web_1 /code/scripts/clean-instance.sh
$ docker exec -i -t opendatacernch_web_1 ./scripts/populate-instance.sh --skip-records
$ docker exec -i -t opendatacernch_web_1 /opt/rh/rh-python36/root/usr/bin/cernopendata fixtures records -f /code/cernopendata/modules/fixtures/data/records/cms-derived-csv-Run2011A.json --mode insert
$ firefox http://localhost/record/5208
```

which works well :+1: but accessing the file itself (`http://localhost/record/5208/files/Zmumu.csv`) fails for now...

Reply:
This should be fixed now. Regarding the problem itself, [we were creating a bucket for records](https://github.com/cernopendata/opendata.cern.ch/blob/a06b6bfdfa603dad2605405e121b5ea128d9c9c2/cernopendata/modules/fixtures/cli.py#L102) (which was correct in the past) but now [records come with buckets by default](https://invenio-records-files.readthedocs.io/en/latest/usage.html#creating-a-record), so we would populate the wrong bucket leaving the Record without files (explanation in commit message too).

The logic for update record needs to be updated too, shall we take this in a different PR?

Reply:
> The logic for update record needs to be updated too, shall we take this in a different PR?

Tested with current code and all works fine (because we clean the buckets and the records already exist, it is fine to create the buckets ourselves).

Reply:
Works locally via browser :+1:  Will need adaptations to `cernopendata-client` though, CC @ParthS007. We can take this later.